src/linter/linterReporting.ts

```diff
@@ -0,0 +1,44 @@
+export const MESSAGES = {
+	SHORT__NO_DIRECT_DATATYPE_ACCESS: "Deprecated access to DataType ('{0}')",
+	DETAILS__NO_DIRECT_DATATYPE_ACCESS: "DataType can be accessed only via the static " +
+	"DataType.getType(\"{0}\") method. " +
+	"{@link topic:00737d6c1b864dc3ab72ef56611491c4 Migrating Deprecated Pseudo Modules}",
```

Comment:
The section of the page is only available with 1.124.0, which is not the default version of https://ui5.sap.com yet, so we should wait for that.

Also, the section is now called "Migrating Access to Pseudo Modules".

Reply:
Done